### PR TITLE
fix media search error reporting and removed clyp private clip restricti...

### PR DESCRIPTION
**Pull request by ScottDowne**
...ons because private videos are behind auth now.

### CODE REVIEW

**Comment by ScottDowne** [comment]
cc @mjschranz

File: public/src/editor/media-gallery-editor.js [hunk 1]
```diff
@@ -343,11 +343,16 @@ define( [ "localized", "util/lang", "util/uri", "util/xhr", "util/keys", "util/m
...
       _addMediaPanel.classList.add( "invalid-field" );
     }, MEDIA_LOAD_TIMEOUT );
-    MediaUtils.getMetaData( data.source, onSuccess, function() {
-      clearTimeout( _cancelSpinner );
-      clearTimeout( _mediaLoadTimeout );
-      deactivateSpinner();
-      pagingSearchCallback( _itemContainers.project, 1 );
+    MediaUtils.getMetaData( data.source, onSuccess, function( errorMessage ) {
+      // Unplayable clip, so display search term pages.
+      if (errorMessage === "This media source is unplayable") {
```
Comment (mjschranz) [inline public/src/editor/media-gallery-editor.js]: You need to check this against whatever the current locale is, so change it to `Localized.get( "This media source is unplayable" )`.

**Comment by mjschranz** [comment]
Looks sane otherwise.

File: public/src/editor/media-gallery-editor.js (hunk 1 above)
Comment (ScottDowne) [inline public/src/editor/media-gallery-editor.js]: Lame, we should translate *right* before spitting it into UI.

Localized error messages in callbacks is kinda silly. Ahh well.

File: public/src/util/mediatypes.js [hunk 2]
```diff
@@ -291,9 +290,6 @@ define( [ "localized", "util/uri", "util/xhr", "json!/api/butterconfig", "jquery
           if ( !respData ) {
             return errorCallback( CLYP_EMBED_UNPLAYABLE );
           }
-          if ( respData.Status !== "Public" ) {
```
Comment (jpatapoff) [inline public/src/util/mediatypes.js]: Looks good to me. As long as the Clyp API returns a 200, the audio file content should be considered importable.

**Pull request merged**